src/Driver/RemoteClient.php

```diff
@@ -658,7 +658,7 @@ private function makeExceptionResponse(Request $request): \Generator
             return yield $this->errorHandler->handleError($status, null, $request);
         } catch (\Throwable $exception) {
             // If the error handler throws, fallback to returning the default HTML error page.
-            $this->logger->error($exception);
+            $this->logger->error($exception->getMessage(), ['exception' => $exception]);
```

Comment:
```suggestion
            $this->logger->error('Error handler threw an exception, falling back to the default HTML error page.', ['exception' => $exception]);
```

Reply:
Whether it's an HTML error page depends on the error handler.

Reply:
? I mean this is a case where the error handler threw so I don't think it depends on it anymore.